prisma/schema.prisma

```diff
@@ -36,7 +36,7 @@ model Password {
 model Group {
   id          String  @id @default(uuid())
   name        String
-  description String
+  description String  @db.Text
```

Comment:
What is this? For bigger texts?

Reply:
Yes! 
Prisma's @db.Text attribute maps a String type to a TEXT type in the database, allowing for unlimited length text values.